plugin-maven/src/main/java/com/diffplug/spotless/maven/generic/Replace.java

```diff
@@ -0,0 +1,28 @@
+package com.diffplug.spotless.maven.generic;
+
+import com.diffplug.spotless.FormatterStep;
+import com.diffplug.spotless.generic.ReplaceStep;
+import com.diffplug.spotless.maven.FormatterStepConfig;
+import com.diffplug.spotless.maven.FormatterStepFactory;
+import org.apache.maven.plugins.annotations.Parameter;
+
+public class Replace implements FormatterStepFactory {
+
+	@Parameter
+	private String name;
+
+	@Parameter
+	private String search;
+
+	@Parameter
+	private String replacement;
+
+	@Override
+	public FormatterStep newFormatterStep(FormatterStepConfig config) {
+		if (name == null || search == null ||replacement == null) {
```

Comment:
I bet `search == null ||replacement` is what the formatter is choking on, but there might be others.  `spotlessApply` to the rescue!